### ISSUE TITLE
vb.metatag: unable to load object bundle executable M4L

### ISSUE THREAD

**Issue by sorrenti90** [v7b1/vb-objects#4]
I'm currently utilizing vb.metatag within an M4L device to extract metadata from audio clips. However, I've encountered a small hiccup. When the Max editor is open, vb.metatag functions smoothly extracting artist information as expected. However, when I use it as an M4L device (with the Max editor closed), vb.metatag fails to work receiving this message:  _vb.metatag: unable to load object bundle executable_

![Captura de pantalla 2024-04-26 a la(s) 12 15 44 a m](https://github.com/v7b1/vb-objects/assets/70607240/2af8b4c5-03f4-45d4-81fc-6b5a328fe1ce)

I share my M4L device.
Thanks!

[vbmetatag_issue.amxd.zip](https://github.com/v7b1/vb-objects/files/15126115/vbmetatag_issue.amxd.zip)

**Comment by v7b1** [comment]
Hi, looks like your Ableton Live is bundled with a different (older) Max version than your standard max in application folder. I'm not really an Ableton user, so might be wrong here, but I think you can change which copy of Max is used by Live in preferences > file folder > Max application. Click on 'browse' and choose your regular Max installation in application folder. Then restart Live and see what happens.

**Comment by sorrenti90** [comment]
Hey there, thank you so much for your answer. I just discovered the problem.
In Ableton Live I'm using the bundled version Max, 8.5.6 version
<img width="358" alt="Captura de pantalla 2024-04-28 a la(s) 4 42 54 p m" src="https://github.com/v7b1/vb-objects/assets/70607240/4cdf0fc4-796d-434e-9cfc-69eecb5a456d">
<img width="208" alt="Captura de pantalla 2024-04-28 a la(s) 4 46 47 p m" src="https://github.com/v7b1/vb-objects/assets/70607240/ba02f370-3abd-403f-b8d8-a53cf31cb07a">

So that wasn't the issue, but thanks to your suggestion, I noticed there's a 'Frameworks' folder within the 'Contents' folder of the Ableton Live app. I simply moved the 'tag.framework' folder from the general Library frameworks folder to the specific Ableton frameworks folder, and voilà, it works!
Thank you so much for your work!

**Comment by sorrenti90** [comment]
Hey there, thank you so much for your answer. I just discovered the problem.
In Ableton Live I'm using the bundled version Max, 8.5.6 version
<img width="358" alt="Captura de pantalla 2024-04-28 a la(s) 4 42 54 p m" src="https://github.com/v7b1/vb-objects/assets/70607240/c118b9e0-4c6d-4c66-a827-68106b7e3fd8">

<img width="208" alt="Captura de pantalla 2024-04-28 a la(s) 4 46 47 p m" src="https://github.com/v7b1/vb-objects/assets/70607240/aef87f4b-2778-4775-a787-26744da5352a">

So that wasn't the issue, but thanks to your suggestion, I noticed there's a 'Frameworks' folder within the 'Contents' folder of the Ableton Live app. I simply moved the 'tag.framework' folder from the general Library frameworks folder to the specific Ableton frameworks folder, and voilà, it works!
Thank you so much for your work!
